packages/ui/package.json

```diff
@@ -5,5 +5,10 @@
   "license": "MIT",
   "devDependencies": {
     "config": "workspace:*"
+  },
+  "dependencies": {
+    "@emotion/react": "^11.8.2",
+    "@emotion/styled": "^11.8.1",
+    "@mui/material": "^5.5.0"
```

Comment:
From the [docs](https://mui.com/getting-started/installation/#npm):

> Please note that [react](https://www.npmjs.com/package/react) >= 17.0.0 and [react-dom](https://www.npmjs.com/package/react-dom) >= 17.0.0 are peer dependencies.

I don't see where we're adding `react`/`react-dom` as peer or full dependencies. Doesn't that cause any issues/warnings when building?